lib.js

```diff
@@ -13,6 +14,9 @@ import { CarReader } from '@ipld/car'
 
 /**
  * @typedef {import('@web3-storage/w3up-client/types').FileLike & { size: number }} FileLike
+ * @typedef {import('@web3-storage/w3up-client/src/types').ListResponse} ListResponse
+ * @typedef {import('@web3-storage/w3up-client/src/types').StoreListResult} StoreListResult
+ * @typedef {import('@web3-storage/w3up-client/src/types').UploadListResult} UploadListResult
```

Comment:
Why `@web3-storage/w3up-client/src/types` not `@web3-storage/w3up-client/types`?

Reply:
oh good question, I suspect I copy/pasted this from somewhere (funny because the path I was looking for was right there in the previous line) - will fix